scripts/versions.sh

```diff
@@ -3,7 +3,8 @@
 # Set up the versions to be used - populate ENV variables only if they are not already populated
 SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.4.12'}
 # Don't export them as they're used in the context of other calls
-AVALANCHEGO_VERSION=${AVALANCHE_VERSION:-'v1.9.11'}
+AVALANCHE_VERSION=${AVALANCHE_VERSION:-'v1.9.11'}
+AVALANCHEGO_VERSION=${AVALANCHEGO_VERSION:-$AVALANCHE_VERSION}
```

Comment:
```suggestion
AVALANCHEGO_VERSION=${AVALANCHEGO_VERSION:-'v1'.9.11'}
```
These are the same thing, can we make them one variable

Reply:
Yeah, I was a little worried about `AVALANCHE_VERSION` being used in other scripts though. My change makes this backwards compatible, so you can change the other scripts one at a time and make sure you don't break anything. I think that would be be better as a follow up PR so the context is a little more clear. 

So how about I do that immediately after this is merged?

Reply:
Sounds good to me